CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to Open Liberty Guides
+
+Anyone can contribute to the Open Liberty guides and we welcome your contributions!
+
+There are multiple ways to contribute: report bugs, fix bugs, contribute code, improve upon documentation, etc. 
+
+## Raising Issues
+
+Please raise any bug reports on the [Open Liberty guide's repository's](https://github.com/OpenLiberty/guide-maven-intro/issues). For new topics, large updates to existing guides, or general suggestions and ideas, report your issue on the [Open Liberty common guides repository](https://github.com/OpenLiberty/guides-common/issues). Be sure to search the list to see if your issue has already been raised.
+
+A good bug report is one that makes it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
+
+## Contributions 
+
+Contributing to the Open Liberty guides is simple. All you need to do is create your own branch to include your contribution(s). If you want to make updates to existing guides, create a new branch and open a pull request to the `qa` (quality assurance) branch. We first merge pull requests to the `qa` branch for staging before we merge to the `master` branch for publishing.
+
+After cloning the guide repository, create a new branch:
+
+```
+git checkout -b [new-branch]
+```
+
+### Text or Content Contribution
+
+If you are contributing text of content changes or trivial code changes via a pull request, sign your work. The sign-off is a line at the end of the commit message that certifies that you wrote it or otherwise have the right to pass it on as an open source patch. The process is simple: if you can certify the [Developer Certificate of Origin (DCO)](https://developercertificate.org)),
```

Comment:
- [ ] not mention "or trivial code changes.."
- [ ] suggest something like 
  - If you are contributing text or content changes and accept the DCO,  sign off your work by adding a sign-off statement in the commit message as following:
`Signed-off-by: John Doe <john.doe@example.com>`